spec/services/store_sp_metadata_in_session_spec.rb

```diff
@@ -27,6 +27,7 @@
         app_session_hash = {
           issuer: 'issuer',
           aal_level_requested: nil,
+          acr_values: nil,
```

Comment:
Note to self: These are currently storing nil. We'll want to implement some tests to ensure we put a value in here.